hack/ci-e2e.sh

```diff
@@ -17,6 +17,21 @@ cd "${REPO_ROOT}" || exit 1
 # BMO_E2E_EMULATOR can be set to either "vbmc" or "sushy-tools"
 BMO_E2E_EMULATOR=${BMO_E2E_EMULATOR:-"sushy-tools"}
 
+case "${GINKGO_FOCUS:-}" in
+  *upgrade*)
+    echo "'upgrade' found in env var GINKGO_FOCUS. Will ONLY run upgrade tests."
+    export GINKGO_FOCUS="upgrade"
```

Comment:
We should not change the variable. If someone has a reason to set this to something more than just upgrade, we just accept it.
```suggestion
    echo "'upgrade' found in env var GINKGO_FOCUS. Will ONLY deploy 'upgrade from' versions of BMO and Ironic."
```